Documentation/En/Index.xyl

```diff
@@ -3,8 +3,166 @@
 <overlay xmlns="http://hoa-project.net/xyl/xylophone">
 <yield id="chapter">
 
-  <p class="warning">This chapter is not yet translated. Contributions are
-  welcomed!</p>
+  <p>Workers frees an application of <strong>heavy</strong> or
+  <strong>blocking</strong> tasks. The <code>Hoa\Worker</code> library creates
+  and manipulates workers effortlessly.</p>
+
+  <h2 id="Table_of_contents">Table of contents</h2>
+
+  <tableofcontents id="main-toc" />
+
+  <h2 id="Introduction" for="main-toc">Introduction</h2>
+
+  <p>It is advisable to read the chapter on
+  <a href="@hack:chapter=Zombie"><code>Hoa\Zombie</code></a> to learn how to
+  transform a processus into a <strong>zombie</strong>. It is the idea behind
+  <code>Hoa\Worker</code> to setup a <strong>server</strong> that will receive
+  the <strong>messages</strong> after the zombie creation. On each received message,
+  an <strong>event</strong> will be issued that we can intercept to execute a
+  <strong>task</strong>.</p>
+  <p>These tasks can be of several kinds such as sending a notification
+  (e.g. by email),  doing important operations on database, handling many files
+  etc. The idea is to <strong>unload</strong> the application from tasks that
+  could <strong>slow it down</strong>. These tasks may be run in
+  <strong>parallel</strong> and must not interfere with the execution of the
+  application.</p>
+  <p>Although the established protocol allows some <strong>integrity</strong>
+  and <strong>security</strong> for messages, it is preferable to do communication
+  from <strong>server to server</strong> and not through the client.
+  <p><code>Hoa\Worker</code> is made of two parts :</p>
+  <ul>
+    <li><code>Hoa\Worker\Backend\Shared</code> to
+    <strong>create</strong> a <strong>shared</strong> worker
+    between <strong>many</strong> processes ;</li>
```

Comment:
```
- ;
+ ,
```
(in English, items are comma separated instead of semi-colon separated as in French).